packages/terra-demographics-banner/CHANGELOG.md

```diff
@@ -2,6 +2,11 @@
 
 ## Unreleased
 
+## 3.59.1 - (January 31, 2023)
+
+* Changed
+  * Minor dependency version bump
```

Comment:
```suggestion
  * Minor dependency version bump of `terra-image`
```

Reply:
Feel free to update the previously done components as well there is no stopping !! It would help us to know the components that got updated in specified version.